jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/servlet/HttpServletQuickFix.java

```diff
@@ -51,14 +52,14 @@ public List<? extends CodeAction> getCodeActions(JavaCodeActionContext context,
         if (parentType != null) {
             // Create code action
             // interface
-            final String TITLE_MESSAGE = "Let ''{0}'' extend ''{1}''";
-            String args[] = { BasicElementLabels.getJavaElementName(parentType.getName()),
-                    BasicElementLabels.getJavaElementName(ServletConstants.HTTP_SERVLET) };
-            ChangeCorrectionProposal proposal = new ExtendClassProposal(MessageFormat.format(TITLE_MESSAGE, args),
-                    context.getCompilationUnit(), parentType, context.getASTRoot(), "jakarta.servlet.http.HttpServlet",
-                    0);
+            String title = Messages.getMessage("LetClassExtend", 
+                    BasicElementLabels.getJavaElementName(parentType.getName()), 
+                    BasicElementLabels.getJavaElementName(ServletConstants.HTTP_SERVLET));
+            ChangeCorrectionProposal proposal = new ExtendClassProposal(title,
```

Comment:
Remove: `import java.text.MessageFormat;`.
No longer used in this file.

Reply:
Ok.